cmd/aks-periscope/aks-periscope.go

```diff
@@ -25,6 +25,14 @@ func main() {
 
 	clusterType := os.Getenv("CLUSTER_TYPE")
 
+	if utils.IsAzureStackCloud() {
+		err := utils.CopyCert()
+		if err != nil {
+			log.Printf("Failed to copy cert for Azure Stack Cloud environment: %+v", err)
+			panic("Error copying cert")
```

Comment:
Let's remove this panic, because this will cause the container to restart over and over again.